docs/cli.md

```diff
@@ -71,21 +71,20 @@ For example,
 
     $ docker-compose run web python manage.py shell
 
-will start the `web` service and then run `manage.py shell` in python.
-Note that by default, linked services will also be started, unless they are
-already running.
+starts the `web` service and then runs `manage.py shell` in python. Note that
+by default, linked services are also started, unless they are already running.
 
 One-off commands are started in new containers with the same configuration as a
```

Comment:
What do you mean by a "one-off" command? In comparison to what? Is there an area in the compose documentation where we are explaining the differences between one-off commands and the XXX commands?

"normal container" implies there are "abnormal containers"  so if compose has the notion of different types of containers we should identify what are the various kinds of containers.  I think here you just mean something like you can start a container with a command that runs and exits or with a long running command like a daemon/service...but we should explain that concept in our compose docs. Where do we have that?